### PR TITLE
fix(suggest): prevent fetch after select

### PR DESCRIPTION
**Pull request by alexdudescu**
When selecting an item after search, the items value was updated,
thus causing an extra empty fetch

### CODE REVIEW

**Review by robertandreimoldoveanu — APPROVED** [review]

File: projects/angular/components/ui-suggest/src/ui-suggest.component.ts [hunk 1]
```diff
@@ -651,6 +652,10 @@ export class UiSuggestComponent extends UiSuggestMatFormFieldDirective
             ),
             this._inputChange$,
         ).pipe(
+            startWith(''),
+            pairwise(),
+            filter(([prev, current]) => !(prev !== '' && current === '')),
```
Comment (anbalase) [inline projects/angular/components/ui-suggest/src/ui-suggest.component.ts:657]: not 100% about this. 
If we wish to trigger a hard reset `this._reset$` this should always pass. Right now it seems to be filtered

Comment (robertandreimoldoveanu) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.ts:657]: Why would we want to fetch new data after a reset?

Comment (anbalase) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.ts:657]: also, we seem to filter out actual queries here.

**Review by anbalase — CHANGES_REQUESTED** [review]
We're solving this issue on the other PR.

File: projects/angular/components/ui-suggest/src/ui-suggest.component.ts (hunk 1 above)
Comment (anbalase) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.ts:657]: @robertandreimoldoveanu That's the point of the reset stream to trigger a new fetch. UiSuggest can be used as a grid filter, there are case where we call this mechanism.

**Pull request closed without merging**